Apps/ValidationTests/CMakeLists.txt

```diff
@@ -76,6 +76,7 @@ target_link_to_dependencies(ValidationTests
     PRIVATE XMLHttpRequest)
 
 if(APPLE AND NOT IOS)
+    target_link_libraries(ValidationTests PUBLIC "-framework MetalKit")
```

Comment:
Does this need to be PUBLIC? Since it's the final target, there should be no one depending on this target right? Does PRIVATE work?

Reply:
I will try but I think it will work.